examples/fineFoods/admin/mui/src/components/product/categoryFilter.tsx

```diff
@@ -16,7 +21,11 @@ export const CategoryFilter: React.FC<ProductItemProps> = ({
 }) => {
     const t = useTranslate();
 
-    const [filterCategories, setFilterCategories] = useState<string[]>([]);
+    const selectedCategories =
+        getDefaultFilter("category.id", filters, "in") ?? [];
+
+    const [filterCategories, setFilterCategories] =
+        useState<string[]>(selectedCategories);
```

Comment:
```suggestion
    const [filterCategories, setFilterCategories] =
        useState<string[]>(getDefaultFilter("category.id", filters, "in") ?? []);
```
maybe we can use like this